### PR TITLE
Motion blur

### PR DESCRIPTION
**Pull request by charleshamesse**
Added motion blur capability
#5

### CODE REVIEW

File: Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp [hunk 1]
```diff
@@ -85,7 +85,7 @@ void AGTRandomObjectPlacer::PostEditChangeProperty(FPropertyChangedEvent& Proper
 void AGTRandomObjectPlacer::PostEditMove(bool bFinished)
 {
     // Force position update of instanced meshes
-    if (GetWorld() && Points.Num() < 1000 || bFinished)
+    if ((GetWorld() && Points.Num() < 1000) || bFinished)
```
Comment (lolleko) [inline Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88]: Hey thanks for the PR.

Could you explain this change?

This seems unrelated to the new motion blur setting.

Comment (charleshamesse) [reply Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88]: Yes that's unrelated indeed, sorry. I needed it to fix this compilation error (using clang 7.0.1):
```
[2/4] Compile Module.UnrealGT.cpp
In file included from /project-root/Plugins/unrealgt/Intermediate/Build/Linux/B4D820EA/UE4Editor/DebugGame/UnrealGT/Module.UnrealGT.cpp:22:
/project-root/Plugins/unrealgt/Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88:20: error: '&&' within '||' [-Werror,-Wlogical-op-parentheses]
    if (GetWorld() && Points.Num() < 1000 || bFinished)
        ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~ ~~
/project-root/Plugins/unrealgt/Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88:20: note: place parentheses around the '&&' expression to silence this warning
    if (GetWorld() && Points.Num() < 1000 || bFinished)
                   ^
        (                                )
1 error generated.
```

Comment (lolleko) [reply Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88]: Okay that I am  fine with leaving this in but it should be

`if (GetWorld() && (Points.Num() < 1000) || bFinished)` instead
                              ^                                                    ^

Comment (lolleko) [reply Source/UnrealGT/Private/Misc/GTRandomObjectPlacer.cpp:88]:
```suggestion
    if (GetWorld() && (Points.Num() < 1000 || bFinished))
```

**Comment by lolleko** [comment]
Thanks!
